src/index.ts

```diff
@@ -4031,9 +4031,10 @@ const shardusSetup = (): void => {
           throw new Error('Stake transaction timestamp is too old')
         }
 
-        if (operatorEVMAccount.operatorAccountInfo?.nominee?.length > 0 ) {
-          throw new Error(`This node is already staked by another account!`)
-        }
+        // No need for this check. We already verify things in verifyStakeTx() above
+        // if (operatorEVMAccount.operatorAccountInfo?.nominee?.length > 0 ) {
+        //   throw new Error(`This node is already staked by another account!`)
+        // }
```

Comment:
Just remove the code, no need to just comment it out because the working functionality is alive an dwell in verifyStakeTx()

Reply:
Initializing verifyResult to false will make every TX apart from stake/unstake fail. This part of the code runs for all transactions. But we only want to run the verification for stake and unstake transactions. Hence verify stake needs to be true by default, and only get set to false of verifyStake/verifyUnstake fail.

I've removed the commented out code.

Reply:
Ah I see, good challenge